aspnetcore/tutorials/signalr-typescript-webpack/sample/3.x/src/index.ts

```diff
@@ -11,19 +11,19 @@ const connection = new signalR.HubConnectionBuilder()
     .build();
 
 connection.on("messageReceived", (username: string, message: string) => {
-    let messageContainer = document.createElement("div");
+    let m = document.createElement("div");
 
-    messageContainer.innerHTML =
+    m.innerHTML =
         `<div class="message-author">${username}</div><div>${message}</div>`;
 
-    divMessages.appendChild(messageContainer);
+    divMessages.appendChild(m);
     divMessages.scrollTop = divMessages.scrollHeight;
 });
 
 connection.start().catch(err => document.write(err));
 
 tbMessage.addEventListener("keyup", (e: KeyboardEvent) => {
-    if (e.keyCode === 13) {
+    if (e.key === 'Enter') {
```

Comment:
Nit: be consistent all throughout the file with either double or single quotes. My personal preference is single quotes because they involve one less keystroke. I'll leave the decision up to you.

Reply:
I updated per your suggestion and tested the script.  Thanks Scott.